merlin/systems/dag/ops/compat.py

```diff
@@ -35,6 +39,10 @@
     import cuml.ensemble as cuml_ensemble
 except ImportError:
     cuml_ensemble = None
+try:
+    from cuml import ForestInference as cuml_fil
```

Comment:
The pattern above with cuml.ensemble and treelite.sklearn was to avoid an error with only importing cuml or treelite. However it might work ok with this.